hack/tools/janitor/main.go

```diff
@@ -74,7 +77,9 @@ func run(ctx context.Context) error {
 	log := ctrl.LoggerFrom(ctx)
 
 	log.Info("Configured settings", "dry-run", dryRun)
-	log.Info("Configured settings", "folders", vsphereFolders)
+	log.Info("Configured settings", "folder", vsphereFolders)
+	log.Info("Configured settings", "vm-folder", vsphereVMFolders)
+	log.Info("Configured settings", "resource-pool", vsphereResourcePools)
```

Comment:
Should this be folders, vm-folders, ..?

Reply:
I named it the same as the parameter is, because you do `-folder "foo" -folder "bar"` which den results in a `[]string{"foo", "bar"}`. But I'm okay with naming it *s :-) 👍

Reply:
Fine either way for me